dev/PushNotifications/PushNotificationManager.h

```diff
@@ -4,6 +4,23 @@
 #pragma once
 #include "Microsoft.Windows.PushNotifications.PushNotificationManager.g.h"
 
+namespace winrt::Microsoft::Windows::PushNotifications::implementation
+{
+    struct PushNotificationManager;
```

Comment:
Please remove this. Redfinition is not needed if you move the factory implementation back to its old location at the bottom of the file.